.tool-versions

```diff
@@ -1,4 +1,4 @@
-python 3.12.8 3.11.5 3.10.13 3.9.18
+python 3.12.8
```

Comment:
I don't think this should have changed, as it's used to install the versions required by tox.